CHANGELOG.rst

```diff
@@ -2,6 +2,25 @@
 Changelog
 =========
 
+0.10.1 (YYYY-MM-DD)
+===================
+
+* Now requires ``clear-html >= 0.4.0``.
+
+* Added :func:`zyte_common_items.processors.description_processor` and enabled
+  it for the ``description`` fields.
+
+* Added :func:`zyte_common_items.processors.description_html_processor` and
+  enabled it for the ``descriptionHtml`` fields.
+
+* Added default implementations for the ``description`` and ``descriptionHtml``
+  fields (if one of these is user-defined, another one will use it).
```

Comment:
Is it for ProductPage?

Reply:
Do you mean mentioning this explicitly? As we don't mention the same in the previous changelog entry, about currency fields.

Reply:
The difference is that there are other data types which have description field, e.g. Article and BusinessPlace.

Reply:
#61

Reply:
Right.